assets/js/base/components/form/index.tsx

```diff
@@ -2,14 +2,20 @@
  * External dependencies
  */
 import classnames from 'classnames';
-import PropTypes from 'prop-types';
+import { FormEvent } from 'react';
```

Comment:
Should this be `import type`?

Reply:
It [doesn't matter too much](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html), but it's a little safer and more declarative to use type so I've changed it 👍